pavelib/assets.py

```diff
@@ -184,6 +213,7 @@ def compile_coffeescript(*files):
 @task
 @no_help
 @cmdopts([
+    ('system=', 's', 'System to act on'),
```

Comment:
nit: help should be a bit clearer. Like, `which system's assets to compile` or something like that.